src/openforms/submissions/api/serializers.py

```diff
@@ -282,12 +287,46 @@ def update(self, instance, validated_data):
         return instance
 
     def notify_suspension(self, instance: Submission, email: str):
-        logger.info("TODO: properly implement sending e-mail with magic link")
-        send_mail(
-            _("Your form submission"),
-            "Submission is suspended. Resume here: <link>",
-            from_email=settings.DEFAULT_FROM_EMAIL,
-            recipient_list=[email],
+        token = submission_resume_token_generator.make_token(instance)
+
+        continue_path = reverse(
+            "submissions:resume",
+            kwargs={
+                "token": token,
+                "submission_uuid": instance.uuid,
+            },
+        )
+        continue_url = urljoin(settings.BASE_URL, continue_path)
+
+        days_until_removal = (
+            instance.form.incomplete_submissions_removal_limit
+            or GlobalConfiguration.get_solo().incomplete_submissions_removal_limit
+        )
+        datetime_removed = instance.created_on + timedelta(days=days_until_removal)
+
+        context = {
+            "form_name": instance.form.name,
+            "save_date": timezone.now().date().strftime("%d-%m-%Y"),
+            "expiration_date": datetime_removed.date().strftime("%d-%m-%Y"),
```

Comment:
Please do the formatting in the template instead of in python.  This ensures that the correct timezone is used: https://docs.djangoproject.com/en/2.2/topics/i18n/timezones/#time-zone-aware-output-in-templates